### PR TITLE
SDK will provide latency reports in response

### PR DESCRIPTION
**Pull request by laurenng**
Times how long it takes a query to get a responses. 

Here's how it looks: 
image

### CODE REVIEW

File: src/lockstep/lockstep_api.py [hunk 1]
```diff
@@ -158,5 +161,9 @@ def send_request(self, method: str, path: str, body: object, query_params: objec
             headers["Authorization"] = "Bearer " + self.bearerToken
     
         response = requests.request(method, url, headers=headers)
+
+        execution_time = time.time() - start_time
+        print('Execution time in seconds: ' + str(execution_time))
```
Comment (tspence) [inline src/lockstep/lockstep_api.py]: This is a great start!  Let's add two variables to the LockstepResponse object called `ServerDuration` and `RoundTripTime` similar to how this was done on the Java SDK: https://github.com/Lockstep-Network/lockstep-sdk-java/blob/main/src/main/java/io/lockstep/api/RestRequest.java#L204-L205

Comment (laurenng) [reply src/lockstep/lockstep_api.py]: What is the serverDuration? From looking through the code, I understand that roundTripTime is the total time it takes to return the response. But I'm a little confused what serverDuration is.

File: src/lockstep/lockstep_api.py [hunk 2]
```diff
@@ -142,21 +144,27 @@ def send_request(self, method: str, path: str, body: object, query_params: objec
...
+
+        execution_time = time.time() - start_time
         headers = {"Accept": "application/json",
                    "SdkName": self.sdkName,
                    "SdkVersion": self.sdkVersion,
                    "MachineName": self.machineName,
-                   "ApplicationName": self.applicationName}
+                   "ApplicationName": self.applicationName,
+                   "RoundTripTime": execution_time,
+                   "ServerDuration": self.serverDuration}
```
Comment (tspence) [inline src/lockstep/lockstep_api.py]: `serverDuration` is a value that is received in the response, not sent in the header.

We should be parsing the response to retrieve this value.

**Comment by tspence** [comment]
Also, can you please add the .idea folder to .gitignore so we don't get any settings files with this PR?

File: src/lockstep/lockstep_api.py [hunk 3]
```diff
@@ -156,7 +160,15 @@ def send_request(self, method: str, path: str, body: object, query_params: objec
...
+
         response = requests.request(method, url, headers=headers)
-        return response.json()
+
+        execution_time = time.time() - start_time
+        result = response.json()
+
+        # times are represented in milliseconds
+        result["RoundTripTime"] = execution_time * 1000
+        result["ServerDuration"] = response.elapsed.microseconds / 1000
```
Comment (tspence) [inline src/lockstep/lockstep_api.py]: Close!  I think this is going to be `response.headers["ServerDuration"]` or something similar.

Comment (laurenng) [reply src/lockstep/lockstep_api.py]: Fixed! Thanks for the hint :)

**Comment by tspence** [comment]
LGTM!

**Pull request merged**